packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -163,6 +168,27 @@ const sendPayload = async (
   return _sendPayload(secretKey, webhook, body, contentType);
 };
 
+export const sendPayloadNoBooking = async (
+  secretKey: string | null,
+  triggerEvent: string,
+  createdAt: string,
+  webhook: Pick<Webhook, "subscriberUrl" | "appId" | "payloadTemplate">,
+  data: WebhookDataType["metadata"]
+) => {
+  const { appId } = webhook;
+  let body;
+  if (appId === "zapier") {
+    body = getZapierPayloadNonBooking(data);
+  } else {
+    body = JSON.stringify({
```

Comment:
We are planning to change the Zapier payloads to return the same as for webhooks. So, it would be best to start doing that here already. Let's remove `getZapierPayloadNonBooking`